static/lang/re-en.json

```diff
@@ -2260,6 +2260,12 @@
             "BlessingOfDefiance": {
                 "Prompt": "Choose one saving throw."
             },
+            "BlinkCharge": {
+                "Note": {
+                    "CriticalFailure": "The Strike deals no damage, and you take @Damage[(ceil((@actor.flags.pf2e.blinkCharge - 4)/2))d8[force]] damage.",
+                    "Failure": "The Strike deals no damage, but the target takes @Damage[(ceil((@actor.flags.pf2e.blinkCharge - 4)/2))d8[force]]."
```

Comment:
```suggestion
                    "Failure": "On a failure, the Strike deals no damage, but the target takes @Damage[(ceil((@actor.flags.pf2e.blinkCharge - 4)/2))d8[force]] damage."
```

Reply:
Should probably specify the failure and critical failure, since both notes will display if there's no target.

Reply:
Ah yeah, good call